mods/lord/Blocks/lord_skeleton/init.lua

```diff
@@ -0,0 +1,127 @@
+minetest.mod(function(mod)
+	local S = minetest.get_mod_translator()
+
+	local title_sbmg = S("Remains of ancient miner. Scull and bones")
```

Comment:
Code-Style: в новых файлах и по возможности при правке используем одинарные кавычки (по всему файлу замени)
```suggestion
	local title_sbmg = S('Remains of ancient miner. Scull and bones')
```